linux/SystemdMeter.c

```diff
@@ -48,6 +50,7 @@ static void* dlopenHandle = NULL;
 
 #if !defined(BUILD_STATIC) || defined(HAVE_LIBSYSTEMD)
 static sd_bus* bus = NULL;
+static sd_bus* userbus = NULL;
```

Comment:
Maybe better name them:
```suggestion
static sd_bus* bus_system = NULL;
static sd_bus* bus_user = NULL;
```